src/components/form/button.ts

```diff
@@ -1,9 +1,25 @@
 import styled, { css } from 'styled-components'
-import PropTypes from 'prop-types'
 import { Paddings, Margins } from 'themes/function'
 import device from 'themes/device.js'
 
-export const SharedButtonStyle = css`
+interface ButtonProps extends React.ComponentPropsWithoutRef<'button'>, SharedButtonStyleProps {
```

Comment:
Since our convention is using `type` instead of `interface` let's use `type` as much as possible